<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualTestingConnectionParametrsWidget</class>
 <widget class="QWidget" name="ManualTestingConnectionParametrsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Настройки подключения</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="connectionSettingsLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="interferenceSimulationGroupBox">
       <property name="title">
        <string>Режим имитации помех</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="noInterferenceRadioButton">
          <property name="text">
           <string>Не посылать</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QRadioButton" name="randomCountInterferenceRadioButton">
          <property name="text">
           <string>Случайные (случайное количество байт):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="randomCountInterferenceSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
          <property name="value">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QRadioButton" name="fixedCountInterferenceRadioButton">
          <property name="text">
           <string>Случайные (фиксированное количество байт):</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="fixedCountInterferenceSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="presetInterferenceRadioButton">
          <property name="text">
           <string>Заданные</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="prefixTextLabel">
          <property name="text">
           <string>Префикс (в HEX):</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QLineEdit" name="prefixInterferenceLineEdit">
          <property name="text">
           <string>0023</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="postfixTextLabel">
          <property name="text">
           <string>Постфикс (в HEX):</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLineEdit" name="postfixInterferenceLineEdit">
          <property name="text">
           <string>00FF</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="ethernetGroupBox">
       <property name="title">
        <string>Проверка порта Ethernet</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="ehtIpEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="iplabel">
          <property name="text">
           <string>IP адрес КСУ: </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="ethernetLinkTestButton">
          <property name="text">
           <string>Проверить связь</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QTextEdit" name="ehtLogEdit">
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>313</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
